src/node_contextify.cc

```diff
@@ -527,10 +527,26 @@ void ContextifyContext::PropertySetterCallback(
       !is_function)
     return;
 
-  USE(ctx->sandbox()->Set(context, property, value));
-  if (is_contextual_store || is_function) {
-    args.GetReturnValue().Set(value);
+  Local<Value> desc;
+  bool is_get_set_property = false;
+  if (is_declared_on_sandbox &&
+      ctx->sandbox()
+          ->GetOwnPropertyDescriptor(context, property)
+          .ToLocal(&desc) &&
+      desc->IsObject()) {
+    Local<Object> desc_obj = desc.As<Object>();
+    Isolate* isolate = context->GetIsolate();
+    Local<Name> get = String::NewFromUtf8(isolate, "get").ToLocalChecked();
+    Local<Name> set = String::NewFromUtf8(isolate, "set").ToLocalChecked();
```

Comment:
These should probably be declared and used like the other constant strings in `src/env_properties.h`

Reply:
Good point, I'll move them to it

Reply:
Done

Reply:
@addaleax Sorry for the ping: just to confirm, is my change aligned from what you had in mind?

Reply:
Yes :)